src/components/common/form/message-summary/MessageSummary.jsx

```diff
@@ -4,7 +4,7 @@ import MessageItem from '../message-item/MessageItem';
 const MessageSummary = ({ keys }) => {
   return (
     <div
-      className="govuk-notification-banner"
+      className="govuk-notification-banner mobile-banner"
```

Comment:
Why have we added these classes? This isn't a div that only shows on mobiles. Is the media targeting of the screen size not enough to target the div